capstone/src/main/webapp/eventlisting.html

```diff
@@ -6,19 +6,32 @@
     <link href="eventDesign.css" type="text/css" rel="stylesheet">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body class="text-center">
+    <ul class="nav">
+      <div class="logo">
+        <a href="/index.html">VOTING CALENDAR</a>
+      </div>
+      <li><a href="/index.html">Home</a></li>
+      <li><a href="/auth">Login</a></li>
+    </ul>
+    <div class="welcome">
+      <h1 class="cover-heading">Available Events</h1>
+      <hr>
+      <p class="lead">
+        Please Add The Following Event(s) To Your Calendar
+      </p>
+    </div>
     <div class="holder">
       <div class="top">
-        <h3>Event 1</h3>
+        <h3></h3>
       </div> <! end top -->
-      <p>Information About the Event</p>
+      <div class="text">
+        <h4></h4>
+      </div>
       <div class="bottom"></div> <! end bottom -->
-      <div class="middle">
-        <button type="button" onclick="login();">Login</button>
         <form action="/eventlisting" method="POST" name="list">
```

Comment:
This form isn't nested in the "bottom" div, so it shouldn't be indented -- this is probably what's throwing off the linter (since it makes it look like line 35 is indented incorrectly)

Reply:
Ah okay, I'll change it and see.

Reply:
It passed! Thanks!